plugins/dev-scripts/scripts/lint.js

```diff
@@ -25,21 +26,39 @@ console.log(`Running lint for ${packageJson.name}`);
 
 // Create the eslint engine.
 const eslintConfig = require('@blockly/eslint-config');
-const cli = new CLIEngine({
+const cli = new ESLint({
```

Comment:
Should be updated to eslint

Reply:
What do you mean, the variable above is named `ESLint`?

Reply:
I think cli should be eslint.  We are not using CLIEngine anymore so I don't think the name cli makes sense.

Reply:
Oh ok